pypa/pypa.go

```diff
@@ -0,0 +1,125 @@
+package pypa
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/aquasecurity/vuln-list-update/types"
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"github.com/cheggaaa/pb"
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	pypaDir            = "pypa"
+	securityTrackerURL = "https://github.com/pypa/advisory-db/archive/refs/heads/main.zip"
+	retry              = 3
+	yamlExt            = ".yaml"
+)
+
+type options struct {
+	url   string
+	dir   string
+	retry int
+}
+type option func(*options)
+
+type Pypa struct {
```

Comment:
```suggestion
type PyPA struct {
```

According to their website, PyPA is correct.
https://www.pypa.io/en/latest/